.gitlab-ci.yml

```diff
@@ -349,21 +387,21 @@ build_puppy_agent-deb_x64_arm:
     - source /root/.bashrc && conda activate ddpy3
     - inv deps --verbose --dep-vendor-only --no-checks
   script:
-    - GOOS=linux GOARCH=arm inv -e agent.build --puppy
+    - inv -e agent.build --puppy
+    - $S3_CP_CMD $SRC_PATH/$AGENT_BINARIES_DIR/agent $S3_ARTIFACTS_URI/puppy/agent
 
-# build dogstatsd for deb-x64
-build_dogstatsd-deb_x64:
+# build puppy agent for ARM, to make sure the build is not broken because of build targets
+build_puppy_agent-deb_x64_arm:
```

Comment:
I think the name should not have `x64` in it.

Reply:
I kept it to be homogenous with the other ARM tasks and as the ARM building images have it in their names.

Reply:
Yes, but that's a mistake :)

We should change those images to the open source ones, which don't have that typo anymore. See: https://github.com/DataDog/datadog-agent-buildimages/blob/master/.gitlab-ci.yml#L39

Reply:
Probably a good idea to do it as part of this PR: https://github.com/DataDog/datadog-agent/pull/4654/files

Let's merge this one as it is.